app/src/main/res/layout/covid_layout.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="cl.maleb.testcarsales.ui.covid.CovidViewModel" />
+    </data>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ProgressBar
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:visibility="@{viewModel.setVisibility(viewModel.isShowProgress())}" />
+
+        <LinearLayout
+            android:id="@+id/contentLinear"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:orientation="vertical"
+            android:padding="@dimen/padding_default"
+            android:visibility="@{viewModel.setVisibility(viewModel.isShowContent())}">
+
+            <TextView
+                android:id="@+id/tvDate"
+                style="@style/textHeader"
+                android:text="@{@string/date + ' ' + viewModel.selectedDate}" />
```

Comment:
Just like a suggestion... avoid of concatenate strings directly in the xml, I think it's a better idea use a binding adapter or format the text, save it into a variable and then just use it in the xml